rotkehlchen/chain/ethereum/trades.py

```diff
@@ -174,6 +174,7 @@ def serialize(self) -> Dict[str, Any]:
             'log_index': self.log_index,
             'from_address': self.from_address,
             'to_address': self.to_address,
+            'timestamp': self.timestamp,  # TODO VN: remove
```

Comment:
I've just realised we are not serializing the timestamp here. I've added it for debugging purposes though.

Reply:
okay don't forget to remove it as it's not needed at least the way we do things now